src/Angor/Client/Pages/Create.razor

```diff
@@ -1251,4 +1251,10 @@
             project.Metadata.Picture = spacePhotos[random.Next(spacePhotos.Length)];
         }
     }
+
+    private decimal targetAmountInBTC
```

Comment:
properties such as this we put at the top of the code block near the pother properties